### DISCUSSION TITLE
Final Submission 🥇

### DISCUSSION THREAD

**Discussion by javalosr2004** [hack4impact-calpoly/starter-pack#17] [Submissions]
Submit your completed website here!

No changes after your submission will be reviewed. Link your website _and_ the repository.
Great work! :D

**LATE SUBMISSIONS ARE NOT ACCEPTED / CLOSED ON October 4th, 11:59PM**

**Comment by ljbarker** [comment]
Here's the link to my completed [website!](https://ljbarker.github.io/index.html) The repository can be found [here.](https://github.com/ljbarker/ljbarker.github.io) Thank you Hack4Impact team for your work on the Starter Pack, and I hope to get the chance to join the team!

**Comment by clkefe** [comment]
Here is the [website](https://www.efeclk.com/) and the [repo](https://github.com/clkefe/personal-website). Thank you for the Starter Pack!

**Comment by HenryHorse** [comment]
Here's my [website](https://shayandaijavad.com) and the [repo](https://github.com/HenryHorse/HenryHorse.github.io). Thanks!

**Comment by KallosP** [comment]
My [website](https://kallosp.github.io/) and [repo](https://github.com/KallosP/KallosP.github.io). Thanks for the starter pack!

**Comment by riannalei** [comment]
My [website](https://riannalei.github.io/) and [rep](https://github.com/riannalei/riannalei.github.io.git). Thanks for the Starter pack!

**Comment by munozAndrew** [comment]
here's my [website ](https://munozandrew.github.io/PortfolIoHACK4IMPACT/index.html) and here is my [repo ](https://github.com/munozAndrew/PortfolIoHACK4IMPACT)

**Comment by Rubensaghian** [comment]
Hello, here's my [website](https://rubensaghian.github.io/) and the [repo](https://github.com/Rubensaghian/Rubensaghian.github.io). Thanks for the starter pack and this opportunity.

**Comment by khoawack** [comment]
[website](https://khoawack.com/) & [repo](https://github.com/khoawack/khoawack.github.io)

**Comment by alexsio03** [comment]
[repo](https://github.com/alexsio03/alexsio03.github.io) and [site](https://alexsio03.github.io/)

**Comment by Alec-0dell** [comment]
Had a lot of fun making my [site](https://alec-odell.vercel.app/). Here is my [code](https://github.com/Alec-0dell/Alec-0dell-HomePage).

**Comment by vmagdangal** [comment]
[Repository](https://github.com/vmagdangal/vmagdangal.github.io) | [Website Link](https://vmagdangal.github.io/index.html)
This was fun to work on. Thank you!

**Comment by amandayu255** [comment]
Website: https://amandayu255.github.io/
Repo: https://github.com/amandayu255/amandayu255.github.io

**Comment by dionnegregorio** [comment]
Website: https://dionnegregorio.github.io/
Repository: https://github.com/dionnegregorio/dionnegregorio.github.io

**Comment by Taliavivrett** [comment]
[Here](https://taliavivrett.github.io/) is my website and [here](https://github.com/Taliavivrett/Taliavivrett.github.io) is my repository! Thank you :))

**Comment by jjonathanlauu** [comment]
Here is my [website](https://jjonathanlauu.github.io/)! And here is my [repository](https://github.com/jjonathanlauu/jjonathanlauu.github.io?tab=readme-ov-file)!

Thank you!

**Comment by clngo** [comment]
Website: https://clngo.github.io/
Repo: https://github.com/clngo/clngo.github.io

**Comment by Amend37** [comment]
Here is my [Website](https://amend37.github.io) and here is my [repo: ](https://github.com/Amend37?tab=repositories)

**Comment by tranguyen-grc** [comment]
[Website](https://tranguyen-grc.github.io/tranguyen.github.io/index.html)
[Repository](https://github.com/tranguyen-grc/tranguyen.github.io.git)

This is my first time making a website with HTML and CSS. Thanks for the starter pack!

**Comment by michaelmurrayiv** [comment]
Here is a link to my [website](https://michaelmurrayiv.github.io/index.html) and Github [repo](https://github.com/michaelmurrayiv/michaelmurrayiv.github.io). I appreciated chatting about H4I the other day and look forward to hearing from you!

**Comment by arossi09** [comment]
Heres my [website](https://arossi09.github.io/) and [repo](https://github.com/arossi09/arossi09.github.io). Thank you for the starter pack, it helped me understand more about web development and find out just how fun it is to customize a website (: .

**Comment by samikshakarimbil** [comment]
[My repo](https://github.com/samikshakarimbil/samikshakarimbil.github.io) and [my website](https://samikshakarimbil.github.io/index.html)

**Comment by atkin0** [comment]
Here's my [website](https://atkin0.github.io/) and [repo](https://github.com/atkin0/atkin0.github.io), thanks!

**Comment by reakunen** [comment]
[Here](https://reakunen.github.io/) is my website and [here](https://github.com/reakunen/reakunen.github.io) is my repository! 🫡

**Comment by TylerHBE** [comment]
[Here](https://tylerhbe.github.io/index.html) is a link to my website and [here](https://github.com/TylerHBE/TylerHBE.github.io) is my repository!

**Comment by rachanaraju3** [comment]
This is my [website](https://rachanaraju3.github.io/) and this is my [repo](https://github.com/rachanaraju3/rachanaraju3.github.io). Thanks for the starter pack!

**Comment by jasonyu0705** [comment]
[website](https://jasonyu0705.github.io/) this Is the link to my website 
[Github link](https://github.com/jasonyu0705/jasonyu0705.github.io) this is the URL for the repo
Will continue to update the styling but the starter pack was fun. Thanks H4I team!!

**Comment by Firazaba0369** [comment]
Here is the link to my [website](https://firazaba0369.github.io/index.html) and this is my [repo](https://github.com/Firazaba0369/Firazaba0369.github.io?tab=readme-ov-file#franciscos-personal-website). This project was fun and the instructions were easy to follow. Thanks H4I team!

**Comment by Amogh76** [comment]
[Here](https://amogh76.github.io/) is the link to my website. I worked really hard on it and had a lot of fun. There are a few issues here and there particularly with the footer and navigation bar but I'll do my best to continue to fix those. This was my first website and I'm proud of it. [Here](https://github.com/Amogh76/Amogh76.github.io) is the link to the repository as well. Let me know if there are issues with accessing either. Thank you to the Hack4Impact team for showing me how fun website building can be and I look forward to hopefully doing it again as a member!

**Comment by carboninl** [comment]
Here is my [website](https://carboninl.github.io/index.html)! Repository [here](https://github.com/carboninl/carboninl.github.io/tree/main). I had lots of fun going through the starter pack, and I'm proud of my results--thank you guys! :)

**Comment by Tylermhh** [comment]
Here is my [website](https://tylermhh.github.io/) and [Repo](https://github.com/Tylermhh/tylermhh.github.io)! :D

**Comment by koryyuen** [comment]
Here is my [website](https://koryyuen.github.io/koryyuen/) and [repository](https://github.com/koryyuen/koryyuen)! Moving text around the webpage where I wanted it was a bit confusing, but overall had a pretty enjoyable time making the website.

**Comment by AlexBloomer** [comment]
Here is my [website](https://alexbloomer.github.io/) and here is my [repository](https://github.com/AlexBloomer/AlexBloomer.github.io)! the starter pack was super helpful and fun! Thank you!

**Comment by kevinrutledge** [comment]
Here is my [website](https://kevinrutledge.github.io/). Here is my [repository](https://github.com/kevinrutledge/kevinrutledge.github.io). Thank you for the starter pack. I got carried away and supplemented with more guides online.

**Comment by DannyAM127** [comment]
Thank you Hack4Impact for the opportunity to apply and for the starter pack. It’s been a great learning experience! Here’s my [website](https://dannyam127.github.io/) and the [repository](https://github.com/DannyAM127/DannyAM127.github.io) link!

**Comment by hoikinlam** [comment]
My [website](https://hoikinlam.github.io). My [repository](https://github.com/hoikinlam/hoikinlam.github.io). Thank you for the starter pack! It was fun coding a website from scratch :D .

**Comment by EthanSunjoonYang** [comment]
Here is my [website](https://ethansunjoonyang.github.io/index.html) and my [repository](https://github.com/EthanSunjoonYang/EthanSunjoonYang.github.io). Thank you for the starter pack! It helped refresh my memory and was fun to complete :)

**Comment by rygelfuenti** [comment]
Here is my [website](https://rygelfuenti.github.io/) and my [repository](https://github.com/rygelfuenti/rygelfuenti.github.io). I had a lot of fun building my own website for the first time!  Will definitely start doing more personal projects in the future!

**Comment by colinbyang** [comment]
here is my [website ](coli-nyang.github.io) and my [repository](https://github.com/coli-nyang/coli-nyang.github.io)

**Comment by Edgarddragde** [comment]
here is my [website ](https://EdgardDragde.github.io)and my [repository](https://github.com/Edgarddragde/Edgarddragde.github.io.git). I feel like I still have a lot to learn about CSS styling.

**Comment by Eclypsee** [comment]
[here](https://eclypsee.github.io/) is my site, and my [repo ](https://github.com/Eclypsee/eclypsee.github.io)is also linked on the site.

**Comment by anthonythang1** [comment]
Hi everyone [here](https://anthonythang1.github.io/Desktop/anthonythang1.github.io/) is my website and this is my [repo!](https://github.com/anthonythang1/anthonythang1.github.io/tree/main)

**Comment by ssasukek** [comment]
here is my [website ](ssasukek.github.io)and my [repo](https://github.com/ssasukek/ssasukek.github.io)

**Comment by Deep-Singh14** [comment]
[Website](https://deep-singh14.github.io/)
[Repo](https://github.com/Deep-Singh14/Deep-Singh14.github.io)

**Comment by RyanSchmit** [comment]
Here is my [website](https://ryanschmit.github.io/) and my [repo](https://github.com/RyanSchmit/RyanSchmit.github.io)

**Comment by MekaSid** [comment]
Here is my [website](https://mekasid.github.io/) and my [repo](https://github.com/MekaSid/MekaSid.github.io).

**Comment by blwelsh** [comment]
Here is my [submission!](https://blwelsh.github.io) And the repo is linked [here](https://github.com/blwelsh/blwelsh.github.io)

**Comment by 1kyler1** [comment]
Here is my[Website](https://1kyler1.github.io/index.html) and [repo](https://github.com/1kyler1/1kyler1.github.io)

**Comment by darpank23** [comment]
Here's the link to my completed [website!](https://darpank23.github.io/) The repository for the website can be found [here.](https://github.com/darpank23/darpank23.github.io) Thank you Hack4Impact for the amazing starter pack guide, I hope to be working with everyone soon! Thank you!

**Comment by tomaseuu** [comment]
Here is the link to my completed [website!](https://tomaseuu.github.io/thomas-l/). The repository for the website can be found [here](https://github.com/tomaseuu/thomas-l). I enjoyed and had fun doing the starter pack! I really learned a lot. Thank you so much Hack4Impact for an amazing guide toward HTML and CSS. I hope you enjoy my website and can't wait to be working everyone hopefully soon! THANK YOU AGAIN!

**Comment by bbq27** [comment]
Here is the completed [website](https://bbq27.github.io/) and [repo](https://github.com/bbq27/bbq27.github.io)! Thank you so much and I hope to hear back soon!

**Comment by IsaacPressman** [comment]
Here is the [website](https://isaacpressman.github.io/) and [repo](https://github.com/IsaacPressman/IsaacPressman.github.io.git). Will continue to update the website and customize it! Thank you!

**Comment by StearmanRubey** [comment]
Here is my [Website](https://stearmanrubey.github.io/) and my [Repository](https://github.com/StearmanRubey/StearmanRubey.github.io). Thank you all for the starter pack! It was really fun to learn new things!

**Comment by LambNNot** [comment]
[Website](https://lambnnot.github.io/)
[Repository](https://github.com/LambNNot/LambNNot.github.io)

**Comment by jonathanramirez4479** [comment]
Here's the link to my [website](https://jonathanramirez4479.github.io/).
Here's the [repository](https://github.com/jonathanramirez4479/jonathanramirez4479.github.io).

**Comment by DashP06** [comment]
Here is my [website](https://dashp06.github.io/Hack4Impact_Starter_Project/)
Here is my [repository](https://github.com/DashP06/Hack4Impact_Starter_Project)
The starter pack was really interesting to work through!

**Comment by AdrianS2k** [comment]
Here is a link to my [website](https://adrians2k.github.io/)
Here is my [repository](https://github.com/AdrianS2k/AdrianS2k.github.io.git)
Thank you Hack4Impact for this fun project!

**Comment by mio-naka** [comment]
Here is the [website](https://mio-naka.github.io/) and the [repo](https://github.com/mio-naka/mio-naka.github.io). Thank you to the team for creating the Starter Pack, I enjoyed creating my website :)

**Comment by diegonieves8** [comment]
Website: https://diegonieves8.github.io/
Repository: https://github.com/diegonieves8/diegonieves8.github.io.git

**Comment by VH571** [comment]
Here is my [website](https://vh571.github.io/) and [repo](https://github.com/VH571/VH571.github.io). It was actually a pretty cool starter pack, thank you hack4impact for making this!!

**Comment by camilayeremin** [comment]
Hello everyone!!
my website: https://camilayeremin.github.io
my repo: https://github.com/camilayeremin/camilayeremin.github.io

**Comment by jaxonsprogis** [comment]
Here is my [website](https://jaxonsprogis.github.io/index.html) and my [repository](https://github.com/jaxonsprogis/jaxonsprogis.github.io). I had a really good time making this, I personally enjoyed trying to figure how to add the sliding text in my nav bar the most. Feel free to check my website out and reply with any suggestions! Thank you Hack4Impact for making this really fun and educational!

**Comment by NolanKnievel** [comment]
Here is my [website](https://nolanknievel.github.io/) and [repository](https://github.com/NolanKnievel/NolanKnievel.github.io)! Had a lot of fun making this. Web development is definitely a new skill for me but it's super rewarding to watch your creation come together. I tried to base my website off [this](https://i.etsystatic.com/36067604/r/il/4355d2/4230665308/il_794xN.4230665308_r13v.jpg) movie poster for _Whiplash_. Hope you enjoy :)

**Comment by dmart297** [comment]
Here's my [website](dmart297.github.io) and my [repo](https://github.com/dmart297/dmart297.github.io)! Thank you for the opportunity to make this, it was so fun!

If the link doesn't work the url is: dmart297.github.io

**Comment by ghost** [comment]
Here is my [website](https://njahadi.github.io) and my [repo](https://github.com/njahadi/njahadi.github.io)

**Comment by Velevynn** [comment]
Here's my [website](https://velevynn.github.io/) and my [repo](https://github.com/Velevynn/velevynn.github.io)! Mainly just updates but still always a fun time!

**Comment by ian-ang-zhihan** [comment]
Here's my [website](https://ian-ang-zhihan.github.io/) and my [repo](https://github.com/ian-ang-zhihan/ian-ang-zhihan.github.io). Thanks for the starter pack! It was a lot of fun :)

**Comment by Eclypsee** [comment]
[here](https://eclypsee.github.io/) is my site! it is live and polished! go check it out!. the repo is also on the site. [repo](https://github.com/Eclypsee/eclypsee.github.io)

**Comment by kjdiaz508** [comment]
[Here](https://kjdiaz508.github.io/HFI-starter-packet/) is my site, and [here](https://github.com/kjdiaz508/HFI-starter-packet) is my repo.

**Comment by mattphanm** [comment]
Website: https://mattphanm.github.io/MatthewH4I.github.io/
Repo: https://github.com/mattphanm/MatthewH4I.github.io
Thank you!

**Comment by aabro1325** [comment]
Hello! Here is my [website](https://aabro1325.github.io/) and my [repo](https://github.com/aabro1325/aabro1325.github.io).

**Comment by KevinBeltran23** [comment]
Hey everyone,
here is my [website](https://kevinbeltran23.github.io/resume.html) as well as my [repo](https://github.com/KevinBeltran23/KevinBeltran23.github.io)

**Comment by RandomCyberCoder** [comment]
Here is my [website](https://randomcybercoder.github.io/) and here is my [repo](https://randomcybercoder.github.io/). Thank you Hack4Impact for the starter pack. I found the CSS portion really interesting.

**Comment by stkeopila** [comment]
Here is my [Website](https://stkeopila.github.io/)
Here is my [Repo](https://github.com/stkeopila/stkeopila.github.io)

**Comment by kailaaquino** [comment]
Hi! [Here](https://kailaaquino.github.io/index.html) is my website and my [repo](https://github.com/kailaaquino/kailaaquino.github.io)

**Comment by aramirez234** [comment]
Hello here is my [website](https://aramirez234.github.io/) and my [repo](https://github.com/aramirez234/aramirez234.github.io)

I focused on responsive design for my website. As such, some features can only be seen on smaller screen sizes. I would recommend opening the website on a phone.

**Comment by jsol1s** [comment]
Link to my repo https://github.com/jsol1s/jsoli1s.github.io!

**Comment by LeticiaLeon-Rodriguez** [comment]
Here is my [Website](https://leticialeon-rodriguez.github.io/LeticiaLeonRodriguez.github.io) and my [repository](https://github.com/LeticiaLeon-Rodriguez/LeticiaLeonRodriguez.github.io)! I had such a fun time creating this and really enjoyed working my way through the starter pack!

**Comment by Vishnu-Alachi123** [comment]
Here is my [Website](https://vishnu-alachi123.github.io/) and this is the [repo](https://github.com/Vishnu-Alachi123/Vishnu-Alachi123.github.io). Thank you for the starter pack, I learned a lot and had fun doing it!

**Comment by ltrush** [comment]
[Here](https://luketrusheim.github.io) is my website. This was super informative and easy to follow along. I definitely plan on continuing to add to my website :)

[Here](https://github.com/luketrusheim/luketrusheim.github.io) is the repo.

**Comment by emlai1263** [comment]
My [Website](https://emlai1263.github.io) and [repo](https://github.com/emlai1263/emlai1263.github.io). The starter pack was well organized and super informative!

**Comment by Shonueld** [comment]
Here is my [website](https://shonueld.github.io/index.html) and the [repo](https://github.com/Shonueld/Shonueld.github.io). I had a lot of fun making my own website, styling in the way I wanted, and learning how to use Git/Github!

**Comment by qarteu** [comment]
[here is my website](https://qarteu.github.io/). [and here is the repo](https://github.com/qarteu/qarteu.github.io) Thank you for the starter pack

**Comment by TheGreatKiwi** [comment]
Here is my website. It is still a wip since I found out about this last minute and only had a couple of hours, but I thought I'd submit it anyway: file:///C:/Users/keira/thegreatkiwi.github.io/index.html

**Comment by karinaleee** [comment]
Hi! here is my [website](https://karinaleee.github.io/) and here is my [repo](https://github.com/karinaleee/karinaleee.github.io.git)

**Comment by suesuee** [comment]
Here is my [repo](https://github.com/suesuee/suesuee.github.io) and [website](https://suesuee.github.io/)!

**Comment by aleecodi** [comment]
Here is my [repo](https://github.com/crazzedman/crazzedman.github.io) and [website](https://crazzedman.github.io/).

**Comment by LambNNot** [comment]
These are the deliverables for the starter pack!
[Website](https://lambnnot.github.io/)
[Repository](https://github.com/LambNNot/LambNNot.github.io)
I thought this was really helpful and I learned a lot. I hope to get better at developing websites with everyone soon! Thank you :D

**Comment by quinniedapooh** [comment]
Here is the link to my [website](https://quinniedapooh.github.io/) and [repository](https://github.com/quinniedapooh/quinniedapooh.github.io). 

Thank you so much to the Hack4Impact team for providing this opportunity and the resources to build our website and learn HTML and CSS. I had a wonderful experience and look forward to the decisions on Sunday. Thank you in advance!

**Comment by emi-a-dinh** [comment]
Thank you for giving us this amazing resource, I feel like I really understand and hope to apply the concepts that I learned to Hack4Impact projects!

[Website,](https://e-dinh-6.github.io/index.html) [repo](https://github.com/e-dinh-6/e-dinh-6.github.io?tab=readme-ov-file)

**Comment by justinch4n** [comment]
Here's the link to my [website](https://justinch4n.github.io/index.html) and [repository](https://github.com/justinch4n/justinch4n.github.io)!

Thank you for this awesome starter pack! It was a very good learning opportunity and I hope to be able to use this in Hack4Impact projects.

**Comment by anissasoungpanya** [comment]
https://anissasoungpanya.github.io/
https://github.com/anissasoungpanya/anissasoungpanya.github.io

**Comment by JavierMB03** [comment]
Here is the link to my [website ](https://javiermb03.github.io/)and [repository](https://github.com/JavierMB03/JavierMB03.github.io.git).

This has been a fantastic learning experience, and I am eager to continue improving my skills. Before this, I knew nothing about HTML and web design, but this intro course taught me so much. Thank you, Hack4Impact.

**Comment by johnnyjjue** [comment]
Thank you for this amazing opportunity, this was my first time creating a website ever so there's still some kinks that I need to figure out. Here's the link to my [website](https://johnnyjjue.github.io/index.html) and [repository](https://github.com/johnnyjjue/johnnyjjue.github.io).

**Comment by Ktigas** [comment]
Here are links to my [website](https://ktigas.github.io/index.html) and the [repository](https://github.com/Ktigas/ktigas.github.io). Thank you!

**Comment by allysonstauffer** [comment]
This website was so fun to make! Here is the link to my [website](https://allysonstauffer.github.io/) and [repository](https://github.com/allysonstauffer/allysonstauffer.github.io?tab=readme-ov-file). Thank you for your consideration, and I hope I can join the team!

**Comment by Matthew-Le-027** [comment]
This is the [link](https://matthew-le-027.github.io/index.html) and this is the link to the [repository](https://github.com/Matthew-Le-027/matthew-le-027.github.io). Thanks for taking the time to look at my application and submission!

**Comment by KailuanLiu** [comment]
[website ](kailuanliu.github.io)
[respository](https://github.com/KailuanLiu/kailuanliu.github.io)

**Comment by Himeh12** [comment]
Here's the link to my [website](https://himeh12.github.io/index.html)! Here is the [repo](https://github.com/Himeh12/Himeh12.github.io?tab=readme-ov-file)! Thank you for the starter pack I found it very informative!

**Comment by ncoronac** [comment]
This is my [website](https://ncoronac.github.io/) and this is the link to my [repo](https://github.com/ncoronac/ncoronac.github.io)! Thank you for your consideration, I had a lot of fun creating this website. :)

**Comment by epears04** [comment]
Here is there link to my [website](https://epears04.github.io/) and my [repo](https://github.com/epears04/epears04.github.io)! Thank you for the great in-depth tutorial.

**Comment by zel701** [comment]
This is the website link: https://zel701.github.io/ This is the link for my repository: https://github.com/zel701/zel701.github.io.git/. Thanks for the tutorial! 
PS: Come to check out my cat

**Comment by ivantorriani** [comment]
Here is my [website](https://ivantorriani.github.io/index.html) and the [repository](https://github.com/ivantorriani/ivantorriani.github.io). Thank you for putting together this awesome demo!

**Comment by Anivishy** [comment]
Here is my [website](https://anivishy.github.io/) and here is my [repo](https://github.com/Anivishy/anivishy.github.io?tab=readme-ov-file)

**Comment by patstarzhou** [comment]
Hello!! Here is my [website](https://patstarzhou.github.io/patrick-z.github.io/index.html) and here is my [repository](https://github.com/patstarzhou/patrick-z.github.io) Thank you so much for the start pack, I had so much fun making it and I learned so much!!

**Comment by MatthewBlam** [comment]
Here are the links to my [website](https://matthewblam.github.io) and [repo](https://github.com/MatthewBlam/MatthewBlam.github.io). Thank you!

**Comment by markmcguire011** [comment]
Here's my two links: [website](https://markmcguire011.github.io/index.html) | [repo](https://github.com/markmcguire011/markmcguire011.github.io/blob/main/README.md)

**Comment by jacobponce** [comment]
Here is my [website](https://jacobponce.github.io/index.html) and [repository](https://github.com/jacobponce/jacobponce.github.io)

**Comment by derrickph33** [comment]
Here is the link to my website: [website](https://derrickph33.github.io/), and here is the link to my repo: [repo](https://github.com/derrickph33/derrickph33.github.io)

**Comment by leili2000** [comment]
Here is my [website](https://leili2000.github.io/) and here is my [repository](https://github.com/leili2000/leili2000.github.io). Thank you!

**Comment by shaaronl** [comment]
here are the links to my [website](https://shaaronl.github.io/) and [repo](https://github.com/shaaronl/shaaronl.github.io?tab=readme-ov-file)! thank you for putting together such a thorough + fun starter pack :)

**Comment by bri8679** [comment]
Here is my [website](https://bri8679.github.io/) and here is a link to my [repo](https://github.com/bri8679/bri8679.github.io), Thank you guys for the starter pack

**Comment by wliu4040** [comment]
My [website](https://wliu4040.github.io/index.html) and my [repo](https://github.com/wliu4040/wliu4040.github.io).

**Comment by vkohli244** [comment]
Here's the link to my completed [website](https://vkohli244.github.io) and here is the link to the [repository](https://github.com/vkohli244/vkohli244.github.io.git)

**Comment by AlexsVargas-CS** [comment]
Here is my  website : https://alexsvargas-cs.github.io/

Here is my Repository: https://github.com/AlexsVargas-CS/Alexsvargas-CS.github.io?tab=readme-ov-file

**Comment by jayrajesh27** [comment]
Here's my [website](https://jayrajesh27.github.io/index.html) and my [repo](https://github.com/jayrajesh27/jayrajesh27.github.io.git). Spent wayy too long on this cloning the Apple nav bar, but it was a rewarding experience having gone through it.

**Comment by annahuang28** [comment]
Here's the link to my [website](https://annahuang28.github.io/index.html) and to the [repository](https://github.com/annahuang28/annahuang28.github.io.git). Thank you for the guidance with the starter pack!

**Comment by ryyyvo** [comment]
Here is the link to my [website](https://ryyyvo.github.io/) and the link to my [repository](https://github.com/ryyyvo/ryyyvo.github.io). Thank you!

**Comment by otvalenc** [comment]
Link to website -> [HERE!](https://leinotoo.github.io)
Link to repo -> [HERE!](https://github.com/leinotoo/leinotoo.github.io)

Thank you for this experience!

**Comment by jasmiineng** [comment]
Here's the link to my [website](https://jasmiineng.github.io/) and to the [repository](https://github.com/jasmiineng/jasmiineng.github.io) . Thank you for the starter pack! I learned so much! :)

**Comment by SaurSum8** [comment]
Hello, this is the link to my website: https://saursum8.github.io/

This is the repository: https://github.com/SaurSum8/SaurSum8.github.io

**Comment by chenyizhao14** [comment]
Here's the link to my [website](https://chenyizhao14.github.io/index.html) and to the [repository](https://github.com/chenyizhao14/chenyizhao14.github.io). This was super fun! I've always wanted to do something like this. Thank you for the guidance with the starter pack!

**Comment by andyc88** [comment]
Here is my [website](https://andyc88.github.io/), & this is my [repository](https://github.com/andyc88/andyc88.github.io)!!

**Comment by tdefl** [comment]
Here's the link to my [website](https://tdefl.github.io/personal-portfolio/) and [repository](https://github.com/tdefl/personal-portfolio). Thanks for the tutorial and for taking the time to review my application and submission!

**Comment by hchencp** [comment]
#Hello this is my website -> hchencp.github.io, and here is the respository -> https://github.com/hchencp/hchencp.github.io , thank you.

**Comment by mcmike29** [comment]
Here's the [link](https://mcmike29.github.io/index.html) to my website and here's the [repo](https://github.com/mcmike29/mcmike29.github.io). Thanks so much for super helpful information and teaching in the starterpack! web dev is a lot different but interesting and fun.

**Comment by ewong128** [comment]
Here is [my repository](https://github.com/ewong128/personalweb) and here is [my website](https://ewong128.github.io/personalweb/index.html). Thanks Hack4Impact for the tutorial and resources, I had an adventure making it.

**Comment by taikimorita** [comment]
Here is my finalized website: https://taikimorita.github.io/ and repository: https://github.com/taikimorita/taikimorita.github.io. Thank you so much Hack4Impact for this amazing and informative Starter Pack!

**Comment by westonsam** [comment]
Here's the link to my [repo ](https://github.com/westonsam/westonsam.github.io.git) and my [website](https://westonsam.github.io/). Thanks for the fun starter pack! :)

**Comment by karthik0605** [comment]
Here is the link to my website: [Website](https://karthik0605.github.io)
Here is the link to my repo: [Repo](https://github.com/karthik0605/karthik0605.github.io.git)

**Comment by lalog12** [comment]
[Website link](https://lalog12.github.io/index.html)
[Repository link](https://github.com/lalog12/lalog12.github.io)

Hope you like it!

**Comment by BTK77** [comment]
WEB [https://BTK77.github.io](url) REPO [https://github.com/BTK77/BTK77.github.io](url)

**Reply by BTK77** [reply]
FOUND OUT THIS ONE DOESN'T WORK, POSTED UPDATED ONE.
Apologies.

**Comment by pragtoppo** [comment]
Here's the link to my repo! https://github.com/pragtoppo/pragtoppo.github.io
(Edit: forgot the link to the website itself : https://pragtoppo.github.io/)

**Comment by BTK77** [comment]


**Reply by BTK77** [reply]
[WORKING WEB](https://btk77.github.io/)
[WORKING REPOSITORY](https://github.com/BTK77/BTK77.github.io)

**Comment by KailuanLiu** [comment]
[website](https://kailuanliu.github.io/)
[repository](https://github.com/KailuanLiu/kailuanliu.github.io)

**Comment by lamkin1** [comment]
I shared my website and repo in the "share your website discussion" rather than this one on accident. 
[Website](https://lamkin1.github.io/index.html)
[Repository](https://github.com/lamkin1/lamkin1.github.io?tab=readme-ov-file)

**Comment by akshatk1001** [comment]
I thought I already submitted here, but apparently not. [Here's the link](akshatk1001hack4impact.github.io) to my site.

**Comment by AdrianS2k** [comment]
Here is my repository: [Repository](https://github.com/AdrianS2k/AdrianS2k.github.io.git)
Here is my website: [Website](https://adrians2k.github.io/)

**Comment by geethitarra** [comment]
Hello, below are my repo and website. Hoping to join the team and grow/develop more coding skills!
Here is my website: [website](https://geethitarra.github.io/index.html)
Here is my repository: [repository](https://github.com/geethitarra/geethitarra.github.io)

**Comment by mchn-4305** [comment]
Here is my website: https://mchn-4305.github.io/
Here is my repository: https://github.com/mchn-4305/mchn-4305.github.io.git

**Reply by mchn-4305** [reply]
https://mchn-4305.github.io/

**Comment by ChachaTeraSira** [comment]
Hey All,
It was a fun journey creating my first ever [Personal Website](http://127.0.0.1:5500/index.html), hope I'll get to do this with a team and learn even more.
This is my [REPO](https://github.com/ChachaTeraSira/ChachaTeraSira.github.io) for anyone interested.

**Reply by ChachaTeraSira** [reply]
Sorry, but I think the previous link to the website didn't work, here is the updated [Link to the Website](https://chachaterasira.github.io/)

**Reply by ChachaTeraSira** [reply]
Hey All,
I previously didn't realise that my website is only optimal for my screen ratio, but it's too late for me to change that now. So, when you go on the webpage it is going to be out of proportion. So here are screenshot's from my own display for you all to view. All the hyperlinks and hyperlogo's work.
![Screenshot 2024-10-08 141928](https://github.com/user-attachments/assets/a02c34ab-f2fa-4607-bae1-48409c5362e8)
![Screenshot 2024-10-08 141940](https://github.com/user-attachments/assets/004ad5d3-9a46-4902-930d-37ee7dab38db)
![Screenshot 2024-10-08 141944](https://github.com/user-attachments/assets/a846796b-7a63-4fbf-9bf6-1442e68cc12b)
![Screenshot 2024-10-08 141950](https://github.com/user-attachments/assets/c806396b-2ece-4730-8e52-e7ed3975398c)
![Screenshot 2024-10-08 141956](https://github.com/user-attachments/assets/50133206-395b-4b4a-8e1e-1a219492ba2a)

**Comment by ChristopherYoeurng** [comment]
Website: https://christopheryoeurng.github.io/
Repo: https://github.com/ChristopherYoeurng/christopheryoeurng.github.io
